src/parkapi_sources/converters/kienzler/converter.py

```diff
@@ -9,36 +9,59 @@
 from validataclass.exceptions import ValidationError
 from validataclass.validators import AnythingValidator, DataclassValidator, ListValidator
 
-from parkapi_sources.converters.base_converter.pull import PullConverter
+from parkapi_sources.converters.base_converter.pull import PullConverter, StaticGeojsonDataMixin
 from parkapi_sources.exceptions import ImportParkingSiteException
 from parkapi_sources.models import RealtimeParkingSiteInput, SourceInfo, StaticParkingSiteInput
 
-from .models import KienzlerInput
+from .models import KienzlerGeojsonFeatureInput, KienzlerInput
 
 
-class KienzlerBasePullConverter(PullConverter):
+class KienzlerBasePullConverter(PullConverter, StaticGeojsonDataMixin):
     kienzler_list_validator = ListValidator(AnythingValidator(allowed_types=[dict]))
     kienzler_item_validator = DataclassValidator(KienzlerInput)
+    kienzler_geojson_item_validator = DataclassValidator(KienzlerInput)
+    use_geojson = False
 
     @property
     @abstractmethod
     def config_prefix(self):
         pass
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, use_geojson=False, **kwargs):
         super().__init__(*args, **kwargs)
         self.required_config_keys = [
             f'PARK_API_KIENZLER_{self.config_prefix}_USER',
             f'PARK_API_KIENZLER_{self.config_prefix}_PASSWORD',
             f'PARK_API_KIENZLER_{self.config_prefix}_IDS',
         ]
+        self.use_geojson = use_geojson
+        self.geojson_feature_validator = DataclassValidator(KienzlerGeojsonFeatureInput)
```

Comment:
Why not initializing this outside of __init__? (eg where you most likely will delete `kienzler_geojson_item_validator`)?

Reply:
Done